pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Start nettop monitoring TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Start nettop monitoring TCP sockets from Loopback interfaces:
+
+`nettop -m tcp -t loopback`
```

Comment:
```suggestion
`nettop -m {{tcp}} -t {{loopback}}`
```
We use the token syntax not only for placeholders, but for anything which can be changed

Reply:
Got it, thanks 😄